src/test/java/com/debatetimer/controller/BaseDocumentTest.java

```diff
@@ -0,0 +1,96 @@
+package com.debatetimer.controller;
+
+import static org.mockito.Mockito.when;
+import static org.springframework.restdocs.payload.JsonFieldType.STRING;
+import static org.springframework.restdocs.payload.PayloadDocumentation.fieldWithPath;
+
+import com.debatetimer.domain.member.Member;
+import com.debatetimer.exception.errorcode.ClientErrorCode;
+import com.debatetimer.repository.member.MemberRepository;
+import com.debatetimer.service.member.MemberService;
+import com.debatetimer.service.parliamentary.ParliamentaryService;
+import io.restassured.RestAssured;
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.filter.log.RequestLoggingFilter;
+import io.restassured.filter.log.ResponseLoggingFilter;
+import io.restassured.specification.RequestSpecification;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.server.LocalServerPort;
+import org.springframework.restdocs.RestDocumentationContextProvider;
+import org.springframework.restdocs.RestDocumentationExtension;
+import org.springframework.restdocs.restassured.RestAssuredRestDocumentation;
+import org.springframework.restdocs.restassured.RestAssuredRestDocumentationConfigurer;
+import org.springframework.restdocs.restassured.RestDocumentationFilter;
+import org.springframework.test.context.bean.override.mockito.MockitoBean;
+
+@ExtendWith({RestDocumentationExtension.class, MockitoExtension.class})
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+public abstract class BaseDocumentTest {
+
+    protected static long EXIST_MEMBER_ID = 123L;
+    protected static Member EXIST_MEMBER = new Member(EXIST_MEMBER_ID, "존재하는 멤버");
+
+    protected static RestDocumentationResponse ERROR_RESPONSE = new RestDocumentationResponse()
```

Comment:
[제안]
이거 description이 "에러 메시지" 로 통일되기 보다 실제 반환되는 에러 메시지를 넘겨주도록 protected method로 선언하는 건 어떨까요?

ex)

```suggestion
    protected  RestDocumentationResponse errorResponse(ErrorCode errorCode){
        return new RestDocumentationResponse()
                .responseBodyField(
                        fieldWithPath("message").type(STRING).description(errorCode.getMessage())
                ); 
    }
```

Reply:
1. PR에서 표기하실 때 '변경하고 싶은 부분'을 '앞 뒤 맥락 포함해서' 표시해주세요. 내가 코드를 어떻게 썼는지 한 눈에 안들어오고, 콜리의 의견과 비교하기가 어려워요.
2. 저 부분의 description이 Swagger UI에 안나타나는 부분이네요. 해당 description을 지우는 건 어떨까요?

Reply:
1. 넵. 문제 개선안을 혼자 고민하다보니 당연히 커찬도 해당 문제를 인식하고 있을 줄 알았는데 넘겨짚은 감이 있는 것 같습니다. 유의할게요 😅 
이 코멘트에서 변경하고 싶었던 부분은 api docs에서 ErrorResponse를 "에러 메시지" 통일하는 것보다 각 에러 상황에 맞도록 에러 객체를 지정하기 위해 method를 만들어주는 건 어떨까? 하는 부분이었어요.

2. description이 Swagger UI 에 나타나지 않음은 RestAssured의 실행 결과를 기반으로 adoc 파일이 생성되기 때문이군요! `그럼 지워도 괜찮을 것 같아요.`